source/blog/2016-02-29-up-and-running-with-ovirt-3-6.html.md

```diff
@@ -0,0 +1,310 @@
+---
+title: Up and Running with oVirt 3.6
```

Comment:
The title is a bit misleading, since the post itself is more advanced than "out of the box" and also includes hosted engine and Gluster. Maybe reword to something like "Up and running with oVirt 3.6 Hosted Engine and Gluster Storage"?